pkg/client/responses.go

```diff
@@ -0,0 +1,64 @@
+// SPDX-FileCopyrightText: 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package client
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/graph-gophers/graphql-go"
+	"github.com/sassoftware/event-provenance-registry/pkg/storage"
+)
+
+// Response type is a struct that represents a JSON response with a data field and an optional
+// errors field.
+// @property Data - The `Data` property is of type `interface{}` which means it can hold any type of
+// data. It is used to store the actual data that needs to be returned in the response.
+// @property {string} Errors - The "Errors" property is a string that is used to store any error
+// messages or error information related to the response. It is marked as "omitempty" in the JSON tag,
+// which means that if the value of the "Errors" property is empty or zero, it will be omitted from the
+// JSON
+type Response struct {
+	Data   interface{} `json:"data"`
```

Comment:
I'd do a wide scale replace of `interface{}` to `any`
```suggestion
	Data   any `json:"data"`
```